x-pack/plugins/infra/common/alerting/metrics/types.ts

```diff
@@ -100,14 +101,43 @@ interface BaseMetricExpressionParams {
 export interface NonCountMetricExpressionParams extends BaseMetricExpressionParams {
   aggType: Exclude<Aggregators, Aggregators.COUNT>;
   metric: string;
+  customMetrics: never;
+  equation: never;
+  label: never;
 }
 
 export interface CountMetricExpressionParams extends BaseMetricExpressionParams {
   aggType: Aggregators.COUNT;
   metric: never;
+  customMetrics: never;
+  equation: never;
+  label: never;
```

Comment:
I'm curious. why are these declared as never? Does it mean they won't have any value assigned to